nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/test/java/org/apache/nifi/util/CharacterFilterUtilsTest.java

```diff
@@ -27,7 +27,7 @@ public void filterInvalidCharacters() throws Exception {
         final String filtered = CharacterFilterUtils.filterInvalidXmlCharacters(text);
 
         final String expected = "This is an example with characters that need to be filtered  in it. ";
-        Assert.assertEquals(expected, filtered);
+        Assertions.assertEquals(expected, filtered);
```

Comment:
This is a good opportunity to replace class references with static imports:
```suggestion
        assertEquals(expected, filtered);
```